txscript/data/script_tests.json

```diff
@@ -551,6 +556,370 @@
+["DATA_64 0x7a{64}", "TOALTSTACK FROMALTSTACK 'z'{64} EQUAL", "NONE", "OK", "Fetch DATA_64 from ALTSTACK"],
+["DATA_65 0x7a{65}", "TOALTSTACK FROMALTSTACK 'z'{65} EQUAL", "NONE", "OK", "Fetch DATA_65 from ALTSTACK"],
+["DATA_66 0x7a{66}", "TOALTSTACK FROMALTSTACK 'z'{66} EQUAL", "NONE", "OK", "Fetch DATA_66 from ALTSTACK"],
+["DATA_67 0x7a{67}", "TOALTSTACK FROMALTSTACK 'z'{67} EQUAL", "NONE", "OK", "Fetch DATA_67 from ALTSTACK"],
+["DATA_68 0x7a{68}", "TOALTSTACK FROMALTSTACK 'z'{68} EQUAL", "NONE", "OK", "Fetch DATA_68 from ALTSTACK"],
+["DATA_69 0x7a{69}", "TOALTSTACK FROMALTSTACK 'z'{69} EQUAL", "NONE", "OK", "Fetch DATA_69 from ALTSTACK"],
+["DATA_70 0x7a{70}", "TOALTSTACK FROMALTSTACK 'z'{70} EQUAL", "NONE", "OK", "Fetch DATA_70 from ALTSTACK"],
+["DATA_71 0x7a{71}", "TOALTSTACK FROMALTSTACK 'z'{71} EQUAL", "NONE", "OK", "Fetch DATA_71 from ALTSTACK"],
+["DATA_72 0x7a{72}", "TOALTSTACK FROMALTSTACK 'z'{72} EQUAL", "NONE", "OK", "Fetch DATA_72 from ALTSTACK"],
+["DATA_73 0x7a{73}", "TOALTSTACK FROMALTSTACK 'z'{73} EQUAL", "NONE", "OK", "Fetch DATA_73 from ALTSTACK"],
+["DATA_74 0x7a{74}", "TOALTSTACK FROMALTSTACK 'z'{74} EQUAL", "NONE", "OK", "Fetch DATA_74 from ALTSTACK"],
+["DATA_75 0x7a{75}", "TOALTSTACK FROMALTSTACK 'z'{75} EQUAL", "NONE", "OK", "Fetch DATA_75 from ALTSTACK"],
+["PUSHDATA1 0x4c 0x7a{76}","TOALTSTACK FROMALTSTACK 'z'{76} EQUAL", "NONE", "OK", "Fetch 76 bytes from ALTSTACK"],
+["PUSHDATA2 0x0001 0x7a{256}","TOALTSTACK FROMALTSTACK 'z'{256} EQUAL", "NONE", "OK", "Fetch 256 bytes from ALTSTACK"],
+["FROMALTSTACK", "1", "NONE", "ERR_INVALID_STACK_OPERATION", "Nothing to fetch from ALTSTACK"],
+["1 TOALTSTACK", "FROMALTSTACK 1", "NONE", "ERR_INVALID_ALTSTACK_OPERATION", "alt stack not shared between sig/pubkey"],
+["1", "FROMALTSTACK", "NONE", "ERR_INVALID_STACK_OPERATION"],
+
+["2DROP test coverge"],
+["0 0", "2DROP 1", "NONE", "OK"],
+["OP_2 OP_4", "2DROP 1", "NONE", "OK"],
+["DATA_15 0x7a{15}  DATA_7 0x7a{7}", "2DROP 1", "NONE", "OK"],
+["1", "2DROP 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
+["DATA_7 0x7a{7}", "2DROP 1", "NONE", "ERR_INVALID_STACK_OPERATION", "2DROP requires two inputs"],
+["", "2DROP 1", "NONE", "ERR_INVALID_STACK_OPERATION", "2DROP requires two inputs"],
+
+["2DUP test coverge"],
+["0 1", "2DUP", "NONE", "OK"],
+["OP_2 OP_4", "2DUP 2DROP ADD 6 EQUAL", "NONE", "OK"],
+["OP_2 OP_2", "2DUP ADD ADD ADD 8 EQUAL", "NONE", "OK"],
+["13 14", "2DUP ROT EQUALVERIFY EQUAL", "NONE", "OK"],
+["NOP", "2DUP 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
+["1", "2DUP 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
+["1", "2DUP", "NONE", "ERR_INVALID_STACK_OPERATION"],
+["DATA_7 0x7a{7}", "2DUP 1", "NONE", "ERR_INVALID_STACK_OPERATION", "2DUP requires two inputs"],
+["", "2DUP 1", "NONE", "ERR_INVALID_STACK_OPERATION", "2DUP requires two inputs"],
+["PUSHDATA4 0x01400000 0x62{16385}", "2DUP 1", "NONE", "ERR_SCRIPT_SIZE", "Exceed max allowed script size"],
```

Comment:
The comment on all of these are identical which would make it hard to pinpoint which specific entry failed.   Should include the specific opcode in question in there as well.

I won't comment individually on all the other cases, but there are several.